CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### x/ecocredit
+
+#### State Machine Breaking
+
+- [#1483](https://github.com/regen-network/regen-ledger/pull/1483) Fix amino codec registration for all ecocredit msgs
```

Comment:
This will be released in v4.0.2? Is there a risk that validators see a patch version and think it's safe to update anytime?

This fix needs a coordinated upgrade

Reply:
We'll make sure to let validators know that it's a coordinated upgrade. I think maybe we should branch off to v4.1 for this, let's discuss on the call today?